### ISSUE TITLE
Problem when using fast_html

### ISSUE THREAD

**Issue by DmitryKK** [philss/floki#272]
## Description

When I use Floki with included fast_html I got error. Floki is not working correctly

## To Reproduce

- Using Floki v0.26.0
- Using fast_html v1.0.3
- Using Elixir v1.10.0
- Using Erlang OTP v22.0.7
- With this code:

```elixir
{:ok, %{body: html}} = HTTPoison.get("https://www.youtube.com")
Floki.parse_document(html)
```

I got error:

`{:error, :timeout}`

Then I run code again and got:

```
iex(master_ad8a69fb@127.0.0.1)3> Floki.parse_document(html)

21:48:48.602 [warn]  unhandled handle_info: {#Port<0.657>, {:data, {:eol, "myhtml worker: error: ei_xreceive_msg: Input/output error"}}}
kill: 31257: No such process

21:48:48.665 [error] GenServer FastHtml.Cnode terminating
** (stop) :nodedown
Last message: {:nodedown, :"myhtml_dcda2650@127.0.0.1"}
State: %{addr: :"myhtml_dcda2650@127.0.0.1", exec_path: "/Users/dmitrij/work/tracker_backend/_build/dev/lib/fast_html/priv/myhtml_worker", hostname: "127.0.0.1", pid: 31257, sname: "myhtml_dcda2650", spawn_inactive_timeout: 10000}
 
21:48:48.673 [debug] connecting to myhtml_a025ecb0@127.0.0.1 failed
 
21:48:48.673 [debug] Spawning myhtml_a025ecb0@127.0.0.1
 
21:48:48.692 [debug] c-node is saying: initialising myhtml_a025ecb0@127.0.0.1
 
21:48:48.692 [debug] c-node is saying: connecting to master_ad8a69fb@127.0.0.1

21:48:48.692 [debug] connected to myhtml_a025ecb0@127.0.0.1
 
21:48:49.215 [warn]  'master_ad8a69fb@127.0.0.1' got a corrupted external term from 'myhtml_dcda2650@127.0.0.1' on distribution channel 21430
<<...,131,104,2,119,13,109,121,104,116,109,108,95,119,111,114,107,101,114,104,2,119,2,111,107,108,0,0,0,1,104,3,109,0,0,0,4,104,116,109,108,108,0,0,0,1,104,2,109,0,0,0,4,108,97,110,103,109,0,0,0,5,98,101,45,66,89,108,0,0,0,1,104,2,109,0,0,0,21,100,97,116,97,45,99,97,115,116,45,97,112,105,45,101,110,97,98,108,101,100,109,0,0,0,4,116,114,117,101,106,108,0,0,0,1,104,3,109,0,0,0,4,104,101,97,100,106,108,0,0,0,1,104,3,109,0,0,0,5,115,116,121,108,101,108,0,0,0,1,104,2,109,0,0,0,4,110,97,109,101,109,0,0,0,10,119,119,119,45,114,111,98,111,116,111,106,108,0,0,0,1,109,0,0,3,53,64,102,111,110,116,45,102,97,99,101,123,102,111,110,116,45,102,97,109,105,108,121,58,39,82,111,98,111,116,111,39,59,102,111,110,116,45,115,116,121,108,101,58,105,116,97,108,105,99,59,102,111,110,116,45,119,101,105,103,104,116,58,53,48,48,59,115,114,99,58,108,111,99,97,108,40,39,82,111,98,111,116,111,32,77,101,100,105,117,109,32,73,116,97,108,105,99,39,41,44,108,111,99,97,108,40,39,82,111,98,111,116,111,45,77,101,100,105,117,109,73,116,97,108,105,99,39,41,44,117,114,108,40,47,47,102,111,110,116,115,46,103,115,116,97,116,105,99,46,99,111,109,47,115,47,114,111,98,111,116,111,47,118,49,56,47,75,70,79,106,67,110,113,69,117,57,50,70,114,49,77,117,53,49,83,55,65,67,99,45,67,115,69,46,116,116,102,41,102,111,114,109,97,116,40,39,116,114,117,101,116,121,112,101,39,41,59,125,64,102,111,110,116,45,102,97,99,101,123,102,111,110,116,45,102,97,109,105,108,121,58,39,82,111,98,111,116,111,39,59,102,111,110,116,45,115,116,121,108,101,58,105,116,97,108,105,99,59,102,111,110,116,45,119,101,105,103,104,116,58,52,48,48,59,115,114,99,58,108,111,99,97,108,40,39,82,111,98,111,116,111,32,73,116,97,108,105,99,39,41,44,108,111,99,97,108,40,39,82,111,98,111,116,111,45,73,116,97,108,105,99,39,41,44,117,114,108,40,47,47,102,111,110,116,115,46,103,115,116,97,116,105,99,46,99,111,109,47,115,47,114,111,98,111,116,111,47,118,49,56,47,75,70,79,107,67,110,113,69,117,57,50,70,114,49,77,117,53,49,120,77,73,122,99,46,116,116,102,41,102,111,114,109,97,116,40,39,116,114,117,101,116,121,112,101,39,41,59,125,64,102,111,110,116,45,102,97,99,101,123,102,111,110,116,45,102,97,109,105,108,121,58,39,82,111,98,111,116,111,39,59,102,111,110,116,45,115,116,121,108,101,58,110,111,114,109,97,108,59,102,111,110,116,45,119,101,105,103,104,116,58,52,48,48,59,115,114,99,58,108,111,99,97,108,40,39,82,111,98,111,116,111,32,82,101,103,117,108,97,114,39,41,44,108,111,99,97,108,40,39,82,111,98,111,116,111,45,82,101,103,117,108,97,114,39,41,44,117,114,108,40,47,47,102,111,110,116,115,46,103,115,116,97,116,105,99,46,99,111,109,47,115,47,114,111,98,111,116,111,47,118,49,56,47,75,70,79,109,67,110,113,69,117,57,50,70,114,49,77,117,53,109,120,80,46,116,116,102,41,102,111,114,109,97,116,40,39,116,114,117,101,116,121,112,101,39,41,59,125,64,102,111,110,116,45,102,97,99,101,123,102,111,110,116,45,102,97,109,105,108,121,58,39,82,111,98,111,116,111,39,59,102,111,110,116,45,115,116,121,108,101,58,110,111,114,109,97,108,59,102,111,110,116,45,119,101,105,103,104,116,58,53,48,48,59,115,114,99,58,108,111,99,97,108,40,39,82,111,98,111,116,111,32,77,101,100,105,117,109,39,41,44,108,111,99,97,108,40,39,82,111,98,111,116,111,45,77,101,100,105,117,109,39,41,44,117,114,108,40,47,47,102,111,110,116,115,46,103,115,116,97,116,105,99,46,99,111,109,47,115,47,114,111,98,111,116,111,47,118,49,56,47,75,70,79,108,67,110,113,69,117,57,50,70,114,49,77,109,69,85,57,102,65,66,99,57,46,116,116,102,41,102,111,114,109,97,116,40,39,116,114,117,101,116,121,112,101,39,41,59,125,106,108,0,0,0,1,104,3,109,0,0,0,6,115,99,114,105,112,116,108,0,0,0,1,104,2,109,0,0,0,4,110,97,109,101,109,0,0,0,10,119,119,119,45,114,111,98,111,116,111,106,108,0,0,0,1,109,0,0,0,135,105,102,32,40,100,111,99,117,109,101,110,116,46,102,111,110,116,115,32,38,38,32,100,111,99,117,109,101,110,116,46,102,111,110,116,115,46,108,111,97,100,41,32,123,100,111,99,117,109,101,110,116,46,102,111,110,116,115,46,108,111,97,100,40,34,52,48,48,32,49,48,112,116,32,82,111,98,111,116,111,34,44,32,34,208,145,34,41,59,100,111,99,117,109,101,110,116,46,102,111,110,116,115,46,108,111,97,100,40,34,53,48,48,32,49,48,112,116,32,82,111,98,111,116,111,34,44,32,34,208,145,34,41,59,125,106,108,0,0,0,1,104,3,109,0,0,0,6,115,99,114,105,112,116,106,108,0,0,0,1,109,0,0,7,138,118,97,114,32,121,116,99,115,105,32,61,32,123,103,116,58,32,102,117,110,99,116,105,111,110,40,110,41,32,123,110,32,61,32,40,110,32,124,124,32,39,39,41,32,43,32,39,100,97,116,97,95,39,59,114,101,116,117,114,110,32,121,116,99,115,105,91,110,93,32,124,124,32,40,121,116,99,115,105,91,110,93,32,61,32,123,116,105,99,107,58,32,123,125,44,105,110,102,111,58,32,123,125,125,41,59,125,44,110,111,119,58,32,40,119,105,110,100,111,119,46,112,101,114,102,111,114,109,97,110,99,101,32,38,38,32,119,105,110,100,111,119,46,112,101,114,102,111,114,109,97,110,99,101,46,116,105,109,105,110,103,32,38,38,119,105,110,100,111,119,46,112,101,114,102,111,114,109,97,110,99,101,46,110,111,119,32,38,38,32,119,105,110,100,111,119,46,112,101,114,102,111,114,109,97,110,99,101,46,116,105,109,105,110,103,46,110,97,118,105,103,97,116,105,111,110,83,116,97,114,116,41,32,63,102,117,110,99,116,105,111,110,40,41,32,123,114,101,116,117,114,110,32,119,105,110,100,111,119,46,112,101,114,102,111,114,109,97,110,99,101,46,116,105,109,105,110,103,46,110,97,118,105,103,97,116,105,111,110,83,116,97,114,116,32,43,119,105,110,100,111,119,46,112,101,114,102,111,114,109,97,110,99,101,46,110,111,119,40,41,59,125,32,58,102,117,110,99,116,105,111,110,40,41,32,123,114,101,116,117,114,110,32,40,110,101,119,32,68,97,116,101,40,41,41,46,103,101,116,84,105,109,101,40,41,59,125,44,116,105,99,107,58,32,102,117,110,99,116,105,111,110,40,108,44,32,116,44,32,110,41,32,123,116,105,99,107,115,32,61,32,121,116,99,115,105,46,103,116,40,110,41,46,116,105,99,107,59,118,97,114,32,118,32,61,32,116,32,124,124,32,121,116,99,115,105,46,110,111,119,40,41,59,105,102,32,40,116,105,99,107,115,91,108,93,41,32,123,116,105,99,107,115,91,39,95,39,32,43,32,108,93,32,61,32,40,116,105,99,107,115,91,39,95,39,32,43,32,108,93,32,124,124,32,91,116,105,99,107,115,91,108,93,93,41,59,116,105,99,107,115,91,39,95,39,32,43,32,108,93,46,112,117,115,104,40,118,41,59,125,116,105,99,107,115,91,108,93,32,61,32,118,59,125,44,105,110,102,111,58,32,102,117,110,99,116,105,111,110,40,107,44,32,118,44,32,110,41,32,123,121,116,99,115,105,46,103,116,40,110,41,46,105,110,102,111,91,107,93,32,61,32,118,59,125,44,115,101,116,83,116,97,114,116,58,32,102,117,110,99,116,105,111,110,40,115,44,32,116,44,32,110,41,32,123,121,116,99,115,105,46,105,110,102,111,40,39,121,116,95,115,116,115,39,44,32,115,44,32,110,41,59,121,116,99,115,105,46,116,105,99,107,40,39,95,115,116,97,114,116,39,44,32,116,44,32,110,41,59,125,125,59,40,102,117,110,99,116,105,111,110,40,119,44,32,100,41,32,123,121,116,99,115,105,46,115,101,116,83,116,97,114,116,40,39,100,104,115,39,44,32,119,46,112,101,114,102,111,114,109,97,110,99,101,32,63,32,119,46,112,101,114,102,111,114,109,97,110,99,101,46,116,105,109,105,110,103,46,114,101,115,112,111,110,115,101,83,116,97,114,116,32,58,32,110,117,108,108,41,59,118,97,114,32,105,115,80,114,101,114,101,110,100,101,114,32,61,32,40,100,46,118,105,115,105,98,105,108,105,116,121,83,116,97,116,101,32,124,124,32,100,46,119,101,98,107,105,116,86,105,115,105,98,105,108,105,116,121,83,116,97,116,101,41,32,61,61,32,39,112,114,101,114,101,110,100,101,114,39,59,118,97,114,32,118,78,97,109,101,32,61,32,40,33,100,46,118,105,115,105,98,105,108,105,116,121,83,116,97,116,101,32,38,38,32,100,46,119,101,98,107,105,116,86,105,115,105,98,105,108,105,116,121,83,116,97,116,101,41,63,32,39,119,101,98,107,105,116,118,105,115,105,98,105,108,105,116,121,99,104,97,110,103,101,39,32,58,32,39,118,105,115,105,98,105,108,105,116,121,99,104,97,110,103,101,39,59,105,102,32,40,105,115,80,114,101,114,101,110,100,101,114,41,32,123,121,116,99,115,105,46,105,110,102,111,40,39,112,114,101,114,101,110,100,101,114,39,44,32,49,41,59,118,97,114,32,115,116,97,114,116,84,105,99,107,32,61,32,102,117,110,99,116,1 (truncated)
```

What the problem? Help me plz

**Comment by philss** [comment]
@DmitryKK sorry for the delay. What is your operational system? Could be that we don't have `epmd` running as stated in our README: https://github.com/philss/floki#using-fast_html-as-the-html-parser

**Comment by DmitryKK** [comment]
> @DmitryKK sorry for the delay. What is your operational system? Could be that we don't have `epmd` running as stated in our README: https://github.com/philss/floki#using-fast_html-as-the-html-parser

macOS Mojave. Epmd is launched automatically when start app, isn't it?

**Comment by philss** [comment]
@DmitryKK in theory yes. Do you still experience this problem?
If so, can you check if `epmd` is running with `ps aux | grep epmd`.

**Comment by rinpatch** [comment]
This is a fast_html problem. This was resolved in fast_html 2.0, I've verified the example provided works fine with floki + fast_html 2.0.1

**Comment by philss** [comment]
@rinpatch thank you! :purple_heart:
